tasks/util/patch_minified.js

```diff
@@ -11,5 +12,20 @@ var STR_NEW = 'require("+ a(r) +");';
  *
  */
 module.exports = function patchMinified(minifiedCode) {
-    return minifiedCode.replace(STR_TO_REPLACE, STR_NEW);
+    for(var i = 0; i < ALPHABET.length; i++) {
+        var li = ALPHABET[i];
+
+        for(var j = 0; j < ALPHABET.length; j++) {
+            var lj = ALPHABET[j];
+
+            var MIDDLE = li + '(' + lj + ')';
+
+            var strOld = FRONT + MIDDLE + BACK,
+                strNew = FRONT + ' ' + MIDDLE + ' ' + BACK;
+
+            minifiedCode = minifiedCode.replace(strOld, strNew);
```

Comment:
This is a great tool for these things:

http://scriptular.com/#require%5C(%22%5C%2B(%5Cw)%5C((%5Cw)%5C)%5C%2B%22%5C)%7C%7C%7C%7C%7C%7C%7C%7C%5B%22require(%5C%22%2Bo(s)%2B%5C%22)%22%5D